lib/rage/controller/parameters_wrappers/strong_parameters.rb

```diff
@@ -0,0 +1,11 @@
+require 'singleton'
+
+module ParametersWrappers
+  class StrongParameters
+    include Singleton
```

Comment:
Added singleton here since there is no need to create multiple instances of this class for each controller because it doesn't store any state